pkg/csi_driver/gcfs_driver.go

```diff
@@ -147,8 +147,7 @@ func (driver *GCFSDriver) validateVolumeCapability(c *csi.VolumeCapability) erro
 		return fmt.Errorf("driver only supports mount access type volume capability")
 	}
 	if mountType.FsType != "" {
-		// TODO: uncomment after https://github.com/kubernetes/kubernetes/issues/65122 is fixed
-		// return fmt.Errorf("driver does not support fstype %v", mountType.FsType)
+		return fmt.Errorf("driver does not support fstype %v", mountType.FsType)
```

Comment:
actually can you revert this line.  Looks like the issue hasn't been completely fixed: https://github.com/kubernetes-csi/external-provisioner/issues/328